### DISCUSSION TITLE
Error 413 upload multiple file to S3

### DISCUSSION THREAD

**Discussion by argelismoreno** [adonisjs/core#1298] [Help] [answered]
Hello, i tried to upload multiple image to S3 i did with multipart.file (Streamming), work fine but if i want to to upload a big image like 2mb each one the server return 413 error.

So i look for a solution and make other request to upload the file with a for of loop but the server return the same error.

i can upload 5 image is the size is 1mb or low..

here my example

request.multipart.file('file', {}, async file => {
   const date = new Date();
   let nameImage = `img_${date.getTime()}`;

    // catches validation errors, if any and then throw exception
    const error = file.error();
     if (error.message) {
          throw new Error(error.message)
     };

     // upload file to s3
     if (file.extname == 'jpg' || file.extname == 'png' || file.extname == 'jpeg') {
         const test = await Drive.disk('s3').put(`images/${nameImage}.png`, file.stream);
         array.push(`images/${nameImage}.png`);
     } else {
          throw new Error('Tipo de formato no valido')

     }
});

Sorry for my bad english is not my first language.

i hope you can help me

**Comment by McSneaky** [comment, marked as answer]
Is S3 returning 413 or AdonisJS?
If it's S3 then maybe there is some setting in there? :thinking: 
Also worth to check any proxies you have between client and Adonis (like Nginx) maybe upload size is limited in there

**Reply by argelismoreno** [reply]
I tried to change client_max_body on the nginx but is not working, the server return 413 error.

:/

**Reply by McSneaky** [reply]
Where is 413 coming from, Nginx, AdonisJS or S3?
You can check Nginx logs to check for Nginx, for Adonis and S3 you can use good'ol `console.log()`

**Reply by argelismoreno** [reply]
The proble coming form Nginx, because i did in local server and work fine, i am look the way to solved in the aws server thanks
